indico/modules/search/client/js/components/results/Contribution.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {List, Icon} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import './Contribution.module.scss';
+
+const Contribution = ({title, url, startDt, eventURL, eventTitle, persons}) => (
+  <div styleName="contribution">
+    <List.Header>
+      <a href={url}>{title}</a>
+    </List.Header>
+    <List.Description styleName="description">
+      <List.Item styleName="high-priority">
+        {/* change to something that reminds an event */}
+        <Icon name="calendar check outline" />
+        <a href={eventURL}>{eventTitle}</a>
+      </List.Item>
+      <List.Item>
+        {persons.length !== 0 && (
+          <ul styleName="high-priority">
+            {persons.length > 1 ? <Icon name="users" /> : <Icon name="user" />}
+            {persons.map(item => (
+              <li key={item.id}>{item.title ? `${item.title} ${item.name}` : `${item.name}`}</li>
```

Comment:
<code>\`${item.name}\`</code> doesn't really make sense, you can just use `item.name` directly